docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,58 +1,88 @@
-!!!
-
-## Checking if Dial Number is Available
-@![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ is a newer RPC, so there is a possibility that not all head units will support it. To find out if the RPC is supported by the head unit, check the system capability manager's @![iOS]`hmiCapabilities.phoneCall`!@ @![android,javaSE,javaEE]`hmiCapabilities.isPhoneCallAvailable()`!@ property after the manager has been started successfully.
+## Checking Your App's Permissions
+@![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ is an RPC that is usually restricted by OEMs. As a result, a module may reject your request if your app does not have the correct permissions. Your SDL app may also be restricted to only being allowed to making a phone call when your app is open (i.e. the `hmiLevel` is non-`NONE`) or when it is the currently active app (i.e. the `hmiLevel` is `FULL`). 
 
 @![iOS]
 ##### Objective-C
 ```objc
-[self.sdlManager.systemCapabilityManager updateCapabilityType:SDLSystemCapabilityTypePhoneCall completionHandler:^(NSError * _Nullable error, SDLSystemCapabilityManager * _Nonnull systemCapabilityManager) {
-    BOOL isDialNumberSupported = NO;
-    if (error == nil) {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability.dialNumberEnabled.boolValue;
-    }
-    else {
-        isDialNumberSupported = systemCapabilityManager.hmiCapabilities.phoneCall.boolValue;
+// TODO
+```
+
+##### Swift
+```swift
+let observerId  = sdlManager.permissionManager.addObserver(forRPCs: [SDLRPCFunctionName.dialNumber.rawValue.rawValue], groupType: .any, withHandler: { (allChanges, groupStatus) in
+    // This handler will be called whenever the permission status changes
+    guard groupStatus == SDLPermissionGroupStatus.allowed else { 
+        // Your app does not have permission to send the `DialNumber` request for its current HMI level
+        return 
     }
 
-    <#If making phone calls is supported, send the `DialNumber` RPC#>
-}];
+    // Your app has permission to send the `DialNumber` request for its current HMI level
+})
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+// TODO
+```
+!@
+
+## Checking if the Module Supports Calling a Phone Number
+Since making a phone call is a newer feature, there is a possibility that some legacy modules will reject your request because the module does not support the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request. Once you have successfully connected to the module and you know that your SDL app has been opened, you can check the module's capabilities via the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ as shown in the example below. Please note that you only need to check once if the module supports calling a phone number, however you must wait to perform this check until you know that the SDL app has been opened (i.e. the `hmiLevel` is non-`NONE`). 
```

Comment:
```suggestion
Since making a phone call is a newer feature, there is a possibility that some legacy modules will reject your request because the module does not support the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request. Once you have successfully connected to the module, you can check the module's capabilities via the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ as shown in the example below. Please note that you only need to check once if the module supports calling a phone number, however you must wait to perform this check until you know that the SDL app has been opened (i.e. the `hmiLevel` is non-`NONE`). 
```